checker/tests/command-line/issue618/Makefile

```diff
@@ -1,5 +1,8 @@
 .PHONY: all
 
-all:
+all: clean
```

Comment:
This cleans the project before running the task, which usually isn't helpful.
Adding the `clean` target is good, but I don't think `all` needs to call it.

Reply:
Thanks, removed!